wiki/Game_mode/osu!/en.md

```diff
@@ -100,150 +100,15 @@ If [Auto Pilot](/wiki/Game_modifier/Autopilot) game modifier was used, only the
 
 ## Scoring
 
-*Scoring values can be found in [osu! scoring system](/wiki/Score/osu!).*
+[The osu! score](/wiki/Gameplay/Score/osu!) is a weighted sum of all hit objects' values. It depends on the following:
 
-Scoring section details all the intricacies of scoring, including mathematical formula.
+- [Judgement](/wiki/Gameplay/Judgement/osu!) determines an object's base scoring value (300, 100, 50, or 0 in case of a miss). For [hit circles](/wiki/Hit_object/Hit_circle), well-timed key presses are valued more, both in terms of score and accuracy. [Sliders](/wiki/Hit_object/Slider) and [spinners](/wiki/Hit_object/Spinner) don't have hit windows, but will break active combo, possibly with a miss, if not cleared properly. Objects with higher judgement also provide higher [health](/wiki/Beatmapping/Health) boost.
+- [Accuracy](/wiki/Gameplay/Accuracy#osu!) depends on judgement and shows how precise hits are. Late or early key presses, as well as misses, decrease overall accuracy.
+- [Combo](/wiki/Gameplay/Combo_(score_multiplier)) is a score multiplier: the higher it is, the more the object contributes to the total score. Combo may be [broken](/wiki/Glossary/Combobreak) by a miss or a [slider break](/wiki/Gameplay/Slider_break).
 
-### Grades
+When combo is maintained, the total score grows exponentially. Objects closer to the end of the map are worth orders of magnitude more points than the ones in the beginning, which means that a player will lose way more score on them in case of mistimed hits. As a result, it's possible and very common for a score with lower accuracy to have higher amount of points and beat the score with higher accuracy.
```

Comment:
because score doesn't actually decrease

```suggestion
When combo is maintained, the total score grows exponentially. Objects closer to the end of the map are worth orders of magnitude more points than the ones in the beginning, which means that a player will lose way more potential score on them in case of mistimed hits. As a result, it's possible and very common for a score with lower accuracy to have a higher amount of points and beat a score with higher accuracy.
```